README.adoc

```diff
@@ -14,6 +14,11 @@ The Khronos{regtitle} Vulkan Working Group
 image::images/vulkan_logo.png[Vulkan Logo]
 image::images/khronos_logo.png[Khronos logo]
 
+:lang-jp: lang/jp/
+This document is also available in the following languages: +
+xref:{lang-jp}README-jp.adoc:[Japanese]
```

Comment:
one more quick fix

```suggestion
xref:{lang-jp}README-jp.adoc[Japanese]
```

resolves to `lang/jp/README-jp.adoc:` not `lang/jp/README-jp.adoc`